WooCommerce/Classes/ViewRelated/Products/Add Product/FirstProductCreatedView.swift

```diff
@@ -0,0 +1,103 @@
+import ConfettiSwiftUI
+import SwiftUI
+
+final class FirstProductCreatedHostingController: UIHostingController<FirstProductCreatedView> {
+    init(productURL: URL) {
+        super.init(rootView: FirstProductCreatedView())
+        rootView.onSharingProduct = { [weak self] in
+            guard let self else { return }
+            SharingHelper.shareURL(url: productURL, from: self.view, in: self)
```

Comment:
❓  Is it possible to pass `onCompletion` block to `SharingHelper` and dismiss this screen from it?

Reply:
We don't get a callback when the user submitted any content in the share sheet, so it's not technically possible to handle `onCompletion`.

Reply:
Oh, OK. I thought we could use the`onCompletion` parameter. https://github.com/woocommerce/woocommerce-ios/blob/trunk/WooCommerce/Classes/Tools/SharingHelper.swift#L29